src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: value => {
+      calculator.result += value;
+
+      return calculator;
+    },
```

Comment:
```suggestion
    add(value) {
      this.result += value;
    },
```

Reply:
try so

Reply:
The same: https://drive.google.com/file/d/1wwY0w8EI-ftEV9ls93YprIwYIjqA7b7v/view?usp=sharing